### DISCUSSION TITLE
Mails delivered in SPAM section

### DISCUSSION THREAD

**Discussion by Sebasfc10** [saran-surya/email_auth#80] [Q&A]
I recently implemented the package in my flutter app, I followed all the steps but the emails keep coming to the spam folder, is that normally the case or do I have to modify something in the HTML?

**Comment by saran-surya** [comment]
Hi Can you please check if there are any mailing rules that you have set ?
Also this is an issue with outlook ?

**Reply by saran-surya** [reply]
Marking this stale Feel free to comment back.
